### ISSUE TITLE
use single runtime chunk will cause error

### ISSUE THREAD

**Issue by Austaras** [DanielSchaffer/webpack-babel-multi-target-plugin#56]
set webpack config
``` js
optimization: {
    runtimeChunk: 'single',
}
```
will cause
`ERROR in   TypeError: Cannot read property 'options' of undefined`

**Comment by Austaras** [comment]
This is not fixed, it will cause vendor.js loaded in both preset. @DanielSchaffer can you reopen this?

**Comment by DanielSchaffer** [comment]
In this scenario, does the vendor.js have mixed ES levels? It _might_ be possible that if the stuff that gets put into the vendor chunk is already ES5 / "legacy" compatible, it could just use that for both bundles.

**Comment by Austaras** [comment]
No. I tested it with `typescript-plain` which should generate a vendor.js with only `core-js` and `regenerator-runtime`, it should never be loaded in modern target
